test/build/50_ch-image.bats

```diff
@@ -44,18 +44,30 @@ setup () {
     # slash present.
     ch-image build -t delete/test -f - . << 'EOF'
 FROM 00_tiny
+FROM 00_tiny
+FROM 00_tiny
+FROM 00_tiny
+FROM 00_tiny
```

Comment:
```suggestion
```

I think three stages is plenty for testing purposes.

Reply:
I agree. Relevant here: The three most important numbers in computer science: zero, one, and more than one. 😉